test/lease-manager.ts

```diff
@@ -285,33 +288,15 @@ describe('LeaseManager', () => {
         });
       });
 
-      it('should properly convert any legacy maxExtension values', () => {
-        const maxExtension = 60 * 1000;
```

Comment:
test removal?

Reply:
Yeah, this one is on purpose - I'm removing legacy stuff with the major.